samples/snippets/snippets.py

```diff
@@ -230,6 +230,160 @@ def count_query_with_stale_read(client):
     return [task1, task2, task3]
 
 
+def sum_query_on_kind(client):
+    # [START datastore_sum_aggregation_query_on_kind]
+    # Set up sample entities
+    task1 = datastore.Entity(client.key("Task", 4871162878230528))
+    task2 = datastore.Entity(client.key("Task", 5069834475798528))
+    task3 = datastore.Entity(client.key("Task", 5081353846521856))
```

Comment:
Is there any significance to these numbers?

Reply:
They are randomly generated IDs from the Datastore console. A previous example used `task1`, `task2`, `task3`, but monotonically increasing IDs are an anti-pattern.

Reply:
it might be worth either mentioning that in a comment, or even using random.random() to generate them in the sample  if that would make sense in the context of the code

Reply:
Good point. Updated the code to let client generate the IDs and added a comment about it.